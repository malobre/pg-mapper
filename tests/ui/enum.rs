#[derive(pg_mapper::TryFromRow)]
enum MyEnum {}

fn main() {}
