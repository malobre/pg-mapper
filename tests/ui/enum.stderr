error: `TryFromRow`: enums are not supported
 --> tests/ui/enum.rs:2:1
  |
2 | enum MyEnum {}
  | ^^^^^^^^^^^^^^
